src/services/dragonchain-client/DragonchainClient.ts

```diff
@@ -318,7 +319,7 @@ export class DragonchainClient {
    * @return {Promise<DragonchainTransactionCreateResponse>}
    */
   public createBulkTransaction = async (transactionBulkObject: DragonchainBulkTransactions) => {
```

Comment:
Also I know that this is not part of this specific ticket but the input here is wrong as well
```suggestion
    transactionBulkObject: DragonchainTransactionCreatePayload[]
```